djangocms_references/cms_config.py

```diff
@@ -110,11 +117,32 @@ def version_queryset_modifier(queryset):
     return queryset
 
 
+def unpublish_dependencies(request, version, *args, **kwargs):
+    """Render a partial template with a list of unpublish dependencies"""
+    references = get_all_reference_objects(
+        version.content, draft_and_published=True)
+    all_querysets_empty = all([not q.exists() for q in references])
+    if all_querysets_empty:
+        # When all querysets are empty, handle it the same way as if
+        # there were no querysets at all
+        references = []
+    return render_to_string(
+        'djangocms_references/unpublish_dependencies.html',
+        {'references': references, 'extra_columns': get_extra_columns()}
+    )
+
+
 class ReferencesCMSAppConfig(CMSAppConfig):
     djangocms_references_enabled = True
+    djangocms_versioning_enabled = getattr(
+        settings, "DJANGOCMS_REFERENCES_VERSIONING_ENABLED", True
```

Comment:
This check doesn't seem relevant to the PR title? Or are we doing more than just configuring references for Unpublish? In other words... why didn't we need to know if versioning was enabled for previous references functionality?

Reply:
We are configuring references for Unpublish and the unpublish functionality comes from versioning. Therefore before versioning could be happily disabled and references didn't need to know it even existed. But now that it knows about unpublishing, it needs to know about versioning and needs to tell versioning how to display the dependencies.